kbase-extension/static/kbase/css/kbaseNarrative.css

```diff
@@ -3121,74 +3273,6 @@ button.kb-data-obj {
     color: #929292;
 
 }
-
```

Comment:
This chunk was either renamed and moved above, or deemed unnecessary after refactoring